exercises/practice/pythagorean-triplet/.docs/instructions.append.md

```diff
@@ -1,3 +1,10 @@
 # Instructions append
 
 By default, only `sum` is given to the `triplets` function, but it may optionally also receive `minFactor` and/or `maxFactor`. When these are given, make sure _each_ factor of the triplet is at least `minFactor` and at most `maxFactor`.
+
+<!-- prettier-ignore-start -->
+~~~exercism/advanced
+If you're solving this using the CLI, there's a test case involving large numbers that's currently skipped to avoid timeouts in our online test runner.
```

Comment:
```suggestion
If you're solving this using the CLI, there's a test case involving large numbers that's currently skipped to avoid timeouts in our test runner.
```

Relevant both for browser and cli solutions, but indeed locally it's fine!